core/comment.js

```diff
@@ -10,43 +10,44 @@
  */
 'use strict';
 
-goog.provide('Blockly.Comment');
-
-goog.require('Blockly.browserEvents');
-goog.require('Blockly.Bubble');
-goog.require('Blockly.Css');
-goog.require('Blockly.Events');
+goog.module('Blockly.Comment');
+goog.module.declareLegacyNamespace();
+
+const Block = goog.requireType('Blockly.Block');
+const BlockSvg = goog.requireType('Blockly.BlockSvg');
+const browserEvents = goog.require('Blockly.browserEvents');
+const Bubble = goog.require('Blockly.Bubble');
+const Coordinate = goog.requireType('Blockly.utils.Coordinate');
+const Css = goog.require('Blockly.Css');
```

Comment:
`Blockly.Css.register` is the only method used on this namespace, so this would also be an appropriate candidate for destructuring.
```
const {register} = goog.require('Blockly.Css');
```

Reply:
Done